script/monitor-pgbouncer.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+require 'librato/metrics'
+
+url = ENV['PGBOUNCER_URL'] || 'postgres://pgbouncer@%2Ftmp:6000/pgbouncer'
+interval = ENV['PGBOUNCER_MONITOR_INTERVAL']&.to_i || 10
+librato_source = ENV['DYNO']
+
+Librato::Metrics.authenticate ENV['LIBRATO_USER'], ENV['LIBRATO_TOKEN']
+
+conn = PGconn.open(url)
+
+loop do
+  res  = conn.exec('SHOW POOLS')
+  res.each do |row|
+    keys = %w(cl_active cl_waiting sv_active sv_idle sv_used sv_tested sv_login maxwait)
```

Comment:
This (`keys`) appears to be the same for every row and every loop, so I'm in favor of moving it out 👍